### ISSUE TITLE
cask ... redirected to spam site!

### ISSUE THREAD

**Issue by haakonstorm** [fniephaus/alfred-homebrew#34]
Running workflow "brew info" ... 
"see what you have to do ..."
redirects here:
https://buyinstagramlikes.io/caskroom/

I thougth I had malware inside the Alfred Workflow setup!
Seems it's just domain snatchers... 
Perhaps this workflow could get som sweet, sweet developer update love?

**Comment by fniephaus** [comment]
How did you trigger this? `brew info` works fine on my machine:

![image](https://user-images.githubusercontent.com/2368856/65035634-fb17ed00-d949-11e9-9d0a-39e0655b4188.png)

**Comment by haakonstorm** [comment]
## brew/cask redirect

Sorry, what invoked the error was trying to use `cask` not `brew info`:

<img width="1065" alt="Screenshot of Alfred Preferences (29-09-2019, 00-25-06)" src="https://user-images.githubusercontent.com/193781/65823076-a92b6d00-e24f-11e9-9c30-5d046aa65dca.png">

Hitting `enter` to see what I need to do takes me to a site that is unrelated to `Homebrew`, see more below. 


## from the brew/cask workflow source
 
![Screenshot of Snagit 2019 (18-09-2019, 01-13-19)](https://user-images.githubusercontent.com/193781/65823008-51d8cd00-e24e-11e9-9a05-865ee4809fe1.png)


## caskroom.io

`http://caskroom.io` returns `http 301`:

```
>>> http://caskroom.io

> --------------------------------------------
> 301 Moved Permanently
> --------------------------------------------

Status:	301 Moved Permanently
Code:	301
Server:	nginx
Date:	Sat, 28 Sep 2019 22:19:02 GMT
Content-Type:	text/html; charset=iso-8859-1
Content-Length:	246
Connection:	close
Location:	https://buyinstagramlikes.io/caskroom/



>>> https://buyinstagramlikes.io/caskroom/

> --------------------------------------------
> 200 OK
> --------------------------------------------

Status:	200 OK
Code:	200
Date:	Sat, 28 Sep 2019 22:19:03 GMT
Server:	Apache
Cache-Control:	max-age=0, no-cache, no-store, must-revalidate
Vary:	User-Agent,Accept-Encoding
Last-Modified:	Tue, 24 Sep 2019 08:49:21 GMT
Accept-Ranges:	none
Pragma:	no-cache
Expires:	Mon, 29 Oct 1923 20:30:00 GMT
Content-Length:	17199
Keep-Alive:	timeout=3, max=200
Content-Type:	text/html; charset=UTF-8
Connection:	close
```

**Comment by haakonstorm** [comment]
btw I'm on `macOS Catalina beta`. Latest version of workflow, on latest `Alfred`, `brew` and toolchain.

**Comment by fniephaus** [comment]
@haakonstorm thanks for the update and yes, that link should be removed asap. AFAICT cask no longer needs to be installed as it's part of brew by default. Just to double check, what do you see when you run `brew search --casks`?

**Comment by haakonstorm** [comment]
<img width="915" alt="Screenshot of MailMate (30-09-2019, 17-41-44)" src="https://user-images.githubusercontent.com/193781/65894177-975ddd00-e3a9-11e9-9240-755a17b85b6c.png">

**Comment by fniephaus** [comment]
Oh sorry, that's not supposed to work in Alfred. What happens if you run it in a terminal?

**Comment by haakonstorm** [comment]
Running `brew search --casks` gives what appears to be an exhaustive list of all casks that `brew` can get by enumerating all the configured taps on this particular box:

<img width="424" alt="Screenshot of iTerm2 (01-10-2019, 17-28-46)" src="https://user-images.githubusercontent.com/193781/65976767-ee7ab500-e470-11e9-80e3-1b7bd8b6474d.png">

**Comment by fniephaus** [comment]
This should be fixed in [`v5.0`](https://github.com/fniephaus/alfred-homebrew/releases/tag/v5.0).
